html/js/bootstrap-select.min.js

```diff
@@ -0,0 +1,9 @@
+/*!

```

Comment:
This looks like it's better maintained than select2 which we currently use - any reason you've picked it though as it does mean we now load 2 libraries for the same thing.

Reply:
I didn't know there was another library like this. I'll look into replacing select2.

Reply:
Don't spend too long. From the looks of this one, it might be better to convert to this.

Reply:
I wanted to say I will check where select2 is used in the project and replace with it with bootstrap-select :)

Reply:
Select2 is pretty mature.  I've been building a lot of ajax request handlers for it because previously the code just loads everything in page.  Sometimes causing nightmare page load times.

Will probably will switch to something virtual dom based (vue.js/react) at some point.

Reply:
Yeah we'd discussed this off github and this new one doesn't support ajax anyway so it'll be converted to select.

Reply:
@murrant @laf Ok np I will switch to select2 and try to mimic the bootstrap-select UX